modules/account-lib/test/unit/coin/cspr/transactionBuilder/transactionBuilder.ts

```diff
@@ -11,6 +11,7 @@ describe('Casper Transaction Builder', () => {
     txBuilder.fee({ gasLimit: testData.FEE.gasLimit, gasPrice: testData.FEE.gasPrice });
     txBuilder.source({ address: testData.ACCOUNT_1.publicKey });
     txBuilder.to(testData.ACCOUNT_2.publicKey);
+    txBuilder.transferId(255);
```

Comment:
Add comment about 255 value.

Reply:
That's just a random value. There is no specific reason to use either 255 or any other positive value.